composer.json

```diff
@@ -19,6 +19,7 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
+        "ext-zip": "^1.14.0",
```

Comment:
Required because of https://www.php.net/manual/de/ziparchive.setencryptionindex.php